src/components/Layout/RightPaneLayout/RightPaneLayout.tsx

```diff
@@ -7,10 +7,12 @@ export function RightPaneLayout({
   children,
   paneBgColor = Palette.White,
   paneWidth = 450,
+  rightColumnContent,
 }: {
   children: ReactElement;
   paneBgColor?: Palette;
   paneWidth?: number;
+  rightColumnContent?: ReactElement;
```

Comment:
Suggestion: I'm wondering if we should call this like `defaultContent` or `primaryContent` ... just something to highlight that it's the default/primary content until something else within the pane is opened.

...`defaultPaneContent` ... wdyt?

Reply:
sounds good to me - always open to suggestions, made the change